es/cheatsheet-machine-learning-tips-and-tricks.md

```diff
@@ -1,257 +1,257 @@
 
-&#10230;
+&#10230; [Entrenamiento sobre los conjuntos k-1 y evaluación en el restante, Entrenamiento en observaciones n-p y evaluación en los p restantes
 
 <br>
 
 **31. [Generally k=5 or 10, Case p=1 is called leave-one-out]**
 
-&#10230;
+&#10230; [Generalmente k = 5 o 10, el caso p = 1 se llama dejando uno fuera]
 
 <br>
 
 **32. The most commonly used method is called k-fold cross-validation and splits the training data into k folds to validate the model on one fold while training the model on the k−1 other folds, all of this k times. The error is then averaged over the k folds and is named cross-validation error.**
 
-&#10230;
+&#10230; El método más comúnmente utilizado se denomina validación cruzada k-fold y divide los datos de entrenamiento en k conjuntos para validar el modelo sobre un conjunto mientras se entrena el modelo en los otros k-1 conjuntos, todo esto k veces. El error luego se promedia sobre los k conjuntos y se denomina error de validación cruzada.
 
 <br>
 
 **33. Regularization ― The regularization procedure aims at avoiding the model to overfit the data and thus deals with high variance issues. The following table sums up the different types of commonly used regularization techniques:**
 
-&#10230;
+&#10230; Regularización - el procedimiento de regularización tiene como objetivo evitar que el modelo se sobreajuste a los datos y, por lo tanto, resuelve los problemas de alta varianza. La siguiente tabla resume los diferentes tipos de técnicas de regularización comúnmente utilizadas:
 
 <br>
 
 **34. [Shrinks coefficients to 0, Good for variable selection, Makes coefficients smaller, Tradeoff between variable selection and small coefficients]**
 
-&#10230;
+&#10230; [Reduce los coeficientes a 0, Bueno para la selección de variables, Hace que los coeficientes sean más pequeños, Compensación entre la selección de variables y los coeficientes pequeños]
 
 <br>
 
 **35. Diagnostics**
 
-&#10230;
+&#10230; Diagnóstico
 
 <br>
 
 **36. Bias ― The bias of a model is the difference between the expected prediction and the correct model that we try to predict for given data points.**
 
-&#10230;
+&#10230; Sesgo - el sesgo de un modelo es la diferencia entre la predicción esperada y el modelo correcto que tratamos de predecir para determinados puntos de datos.
 
 <br>
 
 **37. Variance ― The variance of a model is the variability of the model prediction for given data points.**
 
-&#10230;
+&#10230; Varianza - la varianza de un modelo es la variabilidad de la predicción del modelo para puntos de datos dados.
 
 <br>
 
 **38. Bias/variance tradeoff ― The simpler the model, the higher the bias, and the more complex the model, the higher the variance.**
 
-&#10230;
+&#10230; Corrección de sesgo/varianza - cuanto más simple es el modelo, mayor es el sesgo, y cuanto más complejo es el modelo, mayor es la varianza.
```

Comment:
I would say "Relación sesgo/varianza" instead of "Corrección de"

Reply:
I don't agree. It is not a "relation". "tradeoff" is translated literally as _compensación_. So _correción_ seems to have the same meaning here.